client/my-sites/exporter/exporter.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Gridicon from 'components/gridicon';
+import Button from 'components/forms/form-button';
+import AdvancedSettings from 'my-sites/exporter/advanced-settings';
+
+export default React.createClass( {
+	displayName: 'Exporter',
+
+	propTypes: {
+		advancedSettings: PropTypes.shape( {
+			isVisible: PropTypes.bool.isRequired
+		} )
+	},
+
+	render: function() {
+		return (
```

Comment:
Consider extracting all of the props you'll be using in this method and placing it here. Maybe this:

```
const { toggleAdvancedSettings, advancedSettings, toggleSection } = this.props;
```